File: java/src/main/java/com/github/nomad145/purepazaak/java/PurePazaakJava.java
package com.github.nomad145.purepazaak.java;

import playn.java.LWJGLPlatform;

import com.github.nomad145.purepazaak.core.PurePazaak;

public class PurePazaakJava {

  public static void main (String[] args) {
    LWJGLPlatform.Config config = new LWJGLPlatform.Config();
    // use config to customize the Java platform, if needed
    LWJGLPlatform plat = new LWJGLPlatform(config);
    new PurePazaak(plat);
    plat.start();
  }
}

File: java/src/main/java/com/github/nomad145/purepazaak/kotlin/PurePazaakJava.kt
package com.github.nomad145.purepazaak.kotlin;

import com.github.nomad145.purepazaak.core.PurePazaak;
import playn.java.JavaPlatform;
import playn.java.LWJGLPlatform;

fun main(args: Array<String>) {
  val config = JavaPlatform.Config()
  val platform = LWJGLPlatform(config)

  PurePazaak(platform)

  platform.start()
}
